source/docs/articles/local/drupal-drush-command-line-utility.md

```diff
@@ -8,39 +8,20 @@ keywords: Drupal drush, command line, drupal, terminus drush, cli
 ---
 [Drush](http://drush.org) is a command-line interface for Drupal that provides a wide set of utilities for administering and maintaining your site.
 
-Pantheon does not need the settings.php for your site to work, and your Drupal sites do not contain one out of the box. Drush commands require a settings.php file and it's considered a best practice to have one. Simply duplicate the `sites/default/default.settings.php` to `sites/default/settings.php` for Drush to work on a new site.
+Pantheon does not need the settings.php for your site to work, and your Drupal sites do not contain one out of the box. Drush commands require a settings.php file and it's considered a best practice to have one. Simply copy the `sites/default/default.settings.php` to `sites/default/settings.php` for Drush to work on a new site.

```

Comment:
@greg-1-anderson yes I think we should say something about `settings.php`. Most people do not realize drupal 7 does not have a `settings.php` only the `default.settings.php` unlike Drupal 8.

It's subtle but there may be some potential workflow considerations for users.